projects/components/src/dropdown/dropdown.component.spec.ts

```diff
@@ -31,52 +31,53 @@ describe('DropdownComponent', () => {
             ];
             expect(this.dropdown.shouldRenderAsSeparator(0, this.dropdown.items[0])).toEqual(false);
         });
-        it('returns true only when the current item is a separator and the next item is not' + ' a separator', function(
-            this: HasDropdownComponent
-        ): void {
-            const separatorItemIndices = {
-                one: 1,
-                three: 3,
-                four: 4,
-            };
-            this.dropdown.items = [
-                {
-                    textKey: 'action.1',
-                },
-                {
-                    isSeparator: true,
-                },
-                {
-                    textKey: 'action.1',
-                },
-                {
-                    isSeparator: true,
-                },
-                {
-                    isSeparator: true,
-                },
-                {
-                    textKey: 'action.1',
-                },
-            ];
-            expect(this.dropdown.shouldRenderAsSeparator(0, this.dropdown.items[0])).toEqual(false);
-            expect(
-                this.dropdown.shouldRenderAsSeparator(
-                    separatorItemIndices.one,
-                    this.dropdown.items[separatorItemIndices.one]
-                )
-            ).toEqual(true);
-            expect(
-                this.dropdown.shouldRenderAsSeparator(
-                    separatorItemIndices.three,
-                    this.dropdown.items[separatorItemIndices.three]
-                )
-            ).toEqual(false);
-        });
+        it(
+            'returns true only when the current item is a separator and the next item is not' + ' a separator',
```

Comment:
No concat needed?

Reply:
It is needed. Otherwise, the prettier fix is adding the `it` function call back declaration into the same line and it is exceeding the maximum line length?